common/djangoapps/util/testing.py

```diff
@@ -1,9 +1,18 @@
 import sys
 
+from datetime import datetime
+from pytz import UTC
 from mock import patch
 
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
+from openedx.core.djangoapps.course_groups.models import CourseUserGroupPartitionGroup
```

Comment:
The basic premise is that common is the base, and that the other libraries can build upon it. So, ``openedx`` can use ``common``, but ``common`` should not be using ``openedx``. Even worse is having ``common`` or ``openedx`` use either ``lms`` or ``cms`` and that happens a few times too.

So what should you do? It looks like ``ContentGroupTestCase`` and ``TestConditionalContent`` are only ever used in LMS tests, so probably that means they should also live in the LMS. I don't see a good existing file to add them to, so I'd probably just create a new one.